tests/python/relay/test_pass_alpha_equal.py

```diff
@@ -1,7 +1,17 @@
 import tvm
 import numpy as np
 from tvm import relay
-from tvm.relay.ir_pass import alpha_equal
+from tvm.relay import ir_pass
+
+def alpha_equal(x, y):
+    """
+    Wrapper around alpha equality which ensures that
+    the hash function respects equality.
+    """
+    if ir_pass.alpha_equal(x, y):
+        return ir_pass.expr_hash(x) == ir_pass.expr_hash(y)
+    else:
+        return ir_pass.expr_hash(x) != ir_pass.expr_hash(y)
```

Comment:
This seems to imply that our hashes are equal if and only if the expressions are alpha-equal. Are we certain that's true, i.e., that no collisions are true? It's the `else` branch that has me curious

Reply:
In general this probably doesn't hold, but I wanted to test it for these cases to make sure we don't easily collide, happy to change it back to check hashing only when equal.